alfresco-search/src/main/java/org/alfresco/solr/tracker/SolrTrackerScheduler.java

```diff
@@ -171,14 +169,14 @@ public void deleteJobForTrackerInstance(String coreName, Tracker tracker)
         String jobName = this.getJobName(tracker, coreName);
         JobDetail detail = null;
```

Comment:
null initializer is not needed. 

Better, since the reference is initialised on line 173 and then never written, you could even remove this statement and do directly:

```Java
try 
{
     final JobDetail detail = this.scheduler...
...
```   
You could even avoid the conditional statement by using the Optional.ofNullable + a compose method for getting the JobDetail (not a requirement of course, just a suggestion) 

```Java
try 
{
     ofNullable(getJobDetail())
            .map(detail -> (Tracker) detail.getJobDataMap().get(TrackerJob.JOBDATA_TRACKER_KEY))
            .filter(jobTracker -> jobTracker == tracker)
            .ifPresent(detail -> scheduler.deleteJob(new JobKey(jobName, SOLR_JOB_GROUP)));
...
```